realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -123,10 +134,12 @@ public String getName() {
     private final Object errorLock = new Object();
     private volatile ObjectServerError permissionRealmError = null;
     private volatile ObjectServerError managementRealmError = null;
+    private volatile ObjectServerError globalPermissionRealmError = null;
 
     // Cached result of the permission query. This will be filled, once the first PermissionAsyncTask has loaded
     // the result.
-    private RealmResults<Permission> permissions;
+    private RealmResults<Permission> userPermissions;
+    private RealmResults<Permission> defaultPermissions;
```

Comment:
I feel no reason to have two different names internally, if we expose it as `defaultPermission`, i would like to use the unique name `default` here. So, please, rename it back 🥇